servers/services/build.gradle.kts

```diff
@@ -45,6 +45,32 @@ dependencies {
   compileOnly(libs.microprofile.openapi)
   compileOnly(libs.jakarta.validation.api)
 
+  testRuntimeOnly(project(":nessie-server-store"))
+
+  testImplementation(project(":nessie-versioned-persist-store"))
+  testImplementation(project(":nessie-versioned-persist-adapter"))
+  testImplementation(project(":nessie-versioned-persist-serialize"))
+  testImplementation(project(":nessie-versioned-persist-testextension"))
+  testImplementation(project(":nessie-versioned-persist-in-memory"))
+  testImplementation(project(":nessie-versioned-persist-in-memory-test"))
+  testImplementation(project(":nessie-versioned-persist-rocks"))
```

Comment:
Would it make sense to run module-local test only against the in-memory adapter and run specific storage tests in the storage modules by importing the tests into them (as opposed to importing adapters here)? Then, we will not have to enumerate all storage backends here.

Reply:
Not sure whether the then remaining test coverage is good enough, so I'd prefer to keep those here.

Reply:
Test coverage should be the same, just the module where the tests run will be different. I mean, something like moving `ITApiImplsRocks` to `:nessie-versioned-persist-rocks`

Reply:
Test coverage should be the same, just the module where the tests run will be different. I mean, something like moving `ITApiImplsRocks` to `:nessie-versioned-persist-rocks`

Reply:
fair enough